### ISSUE TITLE
circut

### ISSUE THREAD

**Issue by SayedHusain99** [horihiro/esp8266-google-tts#16]
hey can you send the circuit for that code please

**Comment by horihiro** [comment]
No, I can not, if you need an electric circuit works a code on the repo.
I don't know what you want to do.

If you want to confirm if [this sample code](https://github.com/horihiro/esp8266-google-tts/blob/master/examples/SimpleUsage/SimpleUsage.ino) works, only the dev board w/ ESP8266 is needed.

**Comment by SayedHusain99** [comment]
Thank you for your fast respond 
My senior project needs to utilise tts from an application called bylnk by typing the text in the text value and from that to the ardoino code ie your code and then the text will be speech through a circuit 
The problem is that i don't know which circuit will work with your code
I will be very grateful if you can help me with that🌹

**Comment by horihiro** [comment]
This library just returns the mp3 url of TTS from the input text on ESP8266/ESP32 board (ex. https://www.switch-science.com/catalog/6415/ ), so I have no idea to speech the mp3 on the original circuit.
